addons/website/static/src/js/editor/editor.js

```diff
@@ -5,6 +5,14 @@ var weWidgets = require('web_editor.widget');
 var wUtils = require('website.utils');
 
 weWidgets.LinkDialog.include({
```

Comment:
In fact add the website xml file as dependencies otherwise the t-extend may not be applied in all cases.